docs/automated-tests/troubleshooting/use-annotated-run.md

```diff
@@ -6,13 +6,13 @@ position: 3
 ---
 # Use the Annotated Run for Troubleshooting Failures
 
-Quick test execution can be quite handy for debugging purposes, especially when using the feature to toggle on annotations during the run. When enabled, the annotations highlight the target element of each executed step along with a brief message. They are helpful in cases when the sequence of actions cannot be followed and may reveal the reason why a test does not behave as intended.
+The quick test execution can be quite handy for debugging purposes, especially when using the feature to toggle on annotations during the run. When enabled, the annotations highlight the target element of each executed step along with a brief message. They are helpful in cases when the sequence of actions cannot be followed and may reveal the reason why a test does not behave as intended.
 
-The below article describes an end to end scenario, when the quick execution with annotations can be helpful.
+This article describes a complete scenario that demonstrates how the quick test execution with annotations can be helpful.
 
 ## Record the Scenario
 
-For this tutorial we use a sample application. The task is to click on the _Impressive_ radio button and then verify that the word _Impressive_ is displayed in green in the confirmation.
```

Comment:
Avoid using "we, our" in the documentation because the reader cannot correctly identify which group of people hides behind these pronouns.